service/templates/projects.html

```diff
@@ -999,41 +999,6 @@ <h3>Virtual Workstations Commands</h3>
             </div>
           </div>
         </div>
-        <div id="caching-info">
-          <div class="h3">Historical Data Caching Info </div>
-          <div class="form-group">
-            <div class="col-lg-4 col-header">
-              <input type="checkbox" id="disable-stats-cache" ng-model="settingsFormData.disabled_stats_cache"/>
-              <label for="disable-stats-cache">Disable caching</label>
-            </div>
-          </div>
-          <div class="muted small">List of regular expression patterns that specify test filenames to ignore when caching test and task history.</div>
-          <div class="form-group">
-            <div class="col-lg-5 col-header">
-              <label class="control-label">File patterns to ignore</label>
-            </div>
-          </div>
-          <div class="form-group" ng-repeat="(index, pattern) in settingsFormData.files_ignored_from_cache">
-            <div class="col-lg-4">
-              <label class="control-label">[[pattern]]</label>
-            </div>
-            <div class="col-lg-2">
-              <button class="btn btn-default btn-danger" type="button" ng-click="removeCacheIgnoreFile(index)">
```

Comment:
we should probably also remove the corresponding bits from projects.js, like the removeCacheIgnoreFile function.